mvpinternshipapp.client/src/pages/CustomerPage.jsx

```diff
@@ -0,0 +1,151 @@
+import { useState, useEffect } from "react";
+import { Confirm, Message, Modal, ModalHeader, ModalContent, ModalActions, Button, Icon } from "semantic-ui-react";
+
+import api from '../api';
+import DataTable from "../components/DataTable";
+import CustomerDetails from "../components/customer/CustomerDetails";
+
+export function CustomerPage() {
+    const [customers, setCustomers] = useState([]);
+    const [deleteModalState, setDeleteModalState] = useState({ id: undefined, open: false });
+    const [detailModalState, setDetailModalState] = useState({ open: false, locked: false });
+    const [toastState, setToastState] = useState({ hidden: true, success: true, message: "" });
+    const [modalCustomer, setModalCustomer] = useState({});
+    const [loading, setLoading] = useState(true);
+
+    function closeDetailModal() {
```

Comment:
Can you please use the arrow methods (ES6) for functions instead of **functions**? Just for you to familiarise yourself about it.